production/README.md

```diff
@@ -43,6 +43,10 @@ Loki can be run in a single host, no-dependencies mode using the following comma
 You need `go` [v1.10+](https://golang.org/dl/) installed locally.
 
 ```bash
+
+$ go get github.com/grafana/loki
```

Comment:
In a fresh machine with a fresh user I get with the `got get` command:
```
# added $GOPATH in .bashrc
$ echo $GOPATH 
/home/vangelis/go

$ go version
go version go1.11.4 linux/amd64

# no such dir with a fresh user
$ tree -L 5 ~/go/  
/home/vangelis/go/ [error opening dir]

0 directories, 0 files

$ go get github.com/grafana/loki
package github.com/grafana/loki: no Go files in /home/vangelis/go/src/github.com/grafana/loki

$ tree -L 5 ~/go/
/home/vangelis/go/
└── src
    └── github.com
        └── grafana
            └── loki
                ├── Gopkg.lock
                ├── Gopkg.toml
                ├── LICENSE
                ├── Makefile
                ├── README.md
                ├── cmd
                ├── docs
                ├── loki-build-image
                ├── pkg
                ├── production
                ├── tools
                └── vendor

11 directories, 5 files
```

Reply:
That looks about right!  The `go get` is there to put the files in the right place, the error can be ignored.

Reply:
I see, thanks!